### PR TITLE
documentation around AWS SSO integration

### PR DESCRIPTION
**Pull request by sttomm**

### CODE REVIEW

File: docs/meshstack.aws.metering.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ let AwsPlatformKrakenConfiguration =
           Optional Text
       , data-source :
           {-
-          Currently both AWS Cost and Usage Reports and AWS Cost Explorer are supported.
+          Currently both AWS Cost and Uasge Reports and AWS Cost Explorer are supported.
```
Comment (tfelix) [inline docs/meshstack.aws.metering.md:111]: f: typo

**Review by tfelix — APPROVED** [review]

File: docs/meshstack.aws.index.md [hunk 2]
```diff
@@ -243,29 +257,188 @@ In order to roll out CloudFormation Stack Instances in the newly provisioned acc
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
 
-### AWS Role Mapping
+### IAM Configuration
 
-The [project roles](meshcloud.project.md#project-roles) are mapped to user roles in AWS. This mapping is fully customizable. It is also possible to attach a AWS policy automatically to the user role in AWS.
+Currently meshStack supports 2 different ways of integrating AWS IAM with meshStack. The AWS SSO integration is the preferred integration as it
```
Comment (Jelledb) [inline docs/meshstack.aws.index.md:262]:
```suggestion
Currently, meshStack supports 2 different ways of integrating AWS IAM with meshStack, either via AWS SSO or meshIdB (deprecated). The AWS SSO integration is the preferred integration as it
```
As a bonus you could also directly link to these two sections.

File: docs/meshstack.aws.index.md [hunk 3]
```diff
@@ -243,29 +257,188 @@ In order to roll out CloudFormation Stack Instances in the newly provisioned acc
...
+allows using your companies central IdP to log in to AWS. This simplifies integration with meshStack, gives you more control over the AuthN part
```
Comment (Jelledb) [inline docs/meshstack.aws.index.md:263]:
```suggestion
allows using your company's central IdP to log in to AWS. This simplifies integration with meshStack, gives you more control over the AuthN part
```

File: docs/meshstack.aws.index.md [hunk 4]
```diff
@@ -243,29 +257,188 @@ In order to roll out CloudFormation Stack Instances in the newly provisioned acc
...
+and improves UX for endusers when logging in to AWS.
```
Comment (Jelledb) [inline docs/meshstack.aws.index.md:264]:
```suggestion
and improves UX for end-users when logging in to AWS.
```

**Review by Jelledb — APPROVED** [review]

**Pull request merged**